features/providers/check_nonpassport_autogrant.feature

```diff
@@ -45,6 +45,8 @@ Feature: Checking ccms means does NOT auto grant for non passported applications
 
     Then I choose "legal-aid-application-has-dependants-field"
     Then I click 'Save and continue'
+    Then I should be on a page showing 'Check your answers'
```

Comment:
Should we be more specific about which page (e.g `check_income_answers`)?

Reply:
Will update this too